CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -177,31 +177,31 @@ private IntPtr SourceHook(IntPtr hwnd, int msg, IntPtr wParam, IntPtr lParam, re
             switch ((WM)msg)
             {
                 case WM.IME_SETCONTEXT:
-                {
-                    OnImeSetContext(hwnd, (uint)msg, wParam, lParam);
-                    handled = true;
-                    break;
-                }
+                    {
```

Comment:
Revert unnessicary formatting changes please

Reply:
Updated. Sorry to bother you.